tests/Main.hs

```diff
@@ -36,8 +36,11 @@ import SocketUtils (isPortOpen, simpleSockAddr)
 consulPort :: PortNumber
 consulPort = 18500
 
+dc1 :: Datacenter
+dc1 = Datacenter "dc1"
```

Comment:
Why does `"dc1"` work out of the box, is that Consul's default datacenter name?

Reply:
Yep, you can see it in the output from consul when we run tests.

Reply:
I've applied these updates in the latest PR.